### ISSUE TITLE
Portfolio UI improvement: Adding Sorting

### ISSUE THREAD

**Issue by akreider** [AugurProject/turbo#1518]
On the portfolio page it would be nice to be able to sort your positions by date, resolved status, and possibly by size.

I'm not sure what the current default is.  As when I'm showing both resolved and unresolved positions (in 1.5.0) they appear to not be sorted by date, resolved status, or size.

**Comment by bthaile** [comment]
Agreed there needs to be a sorting bar like on LP page so that user has more control over their positions being displayed.
